src/components/FileInputField/FileInputField.jsx

```diff
@@ -37,20 +37,30 @@ export const FileInputField = React.forwardRef((props, ref) => {
     ...restProps
   } = props;
 
-  const internalInputRef = useRef();
-
-  // We need to have a reference to the input element to be able to call its methods,
-  // but at the same time we want to expose this reference to the parent component for
-  // case someone wants to call input methods from outside the component.
-  useImperativeHandle(ref, () => internalInputRef.current);
-
   const formLayoutContext = useContext(FormLayoutContext);
   const inputGroupContext = useContext(InputGroupContext);
   const translations = useContext(TranslationsContext);
 
   const [selectedFileNames, setSelectedFileNames] = useState([]);
   const [isDragging, setIsDragging] = useState(false);
 
+  const internalInputRef = useRef();
+
+  // We need to have a reference to the input element to be able to call its methods,
+  // but at the same time we want to expose this reference to the parent component for
+  // case someone wants to call input methods from outside the component.
+  useImperativeHandle(
+    ref,
+    () => {
+      const inputEl = internalInputRef?.current ?? {};
```

Comment:
I did not use spread syntax on purpose and ESlint is OK with that as well. Reason is that it would somehow change final shape of the object. If you return it as it is, it look like HTML input in the console. If you change it to the spread syntax, it would return object with bunch of attributes. In the end, it will works the same, but somehow Chrome presents it differently.

I think that this is not necessary to document in the code as it only improves debugablitiy for end user.